.github/dependabot.yml

```diff
@@ -12,24 +12,13 @@ updates:
       - dependency-name: "*"
         update-types: ["version-update:semver-patch"]
 
-  - package-ecosystem: "docker"
-    directory: "/"
-    schedule:
-      interval: "monthly"
-    labels:
-      - "kind/dependencies"
-      - "area/tooling"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-patch"]
-
   - package-ecosystem: "gomod"
     directory: "/internal/mage"
     schedule:
       interval: "monthly"
```

Comment:
@gerhard I wonder if we should reduce this time? We sometimes release more often than this, which could mean we could potentially have multiple releases in between when we could get a dependabot PR.

I'd love to have dependency bumps as part of our release process - maybe we could bump packages at the beginning of each release cycle?

Reply:
I agree with this idea if we gain time merging dependencies! 

It always take a lot of time to merge all these dependencies, specially when we need to rebase because the CI can sometime takes up to 15 minutes to be complete (for 1 process).

If bumping package more often reduce the number of dependencies to bump at each time, then I think it's a great! However, if we still have the same number of bump to do each weak than we had by month, then it's not worth it IMO.